griptape/artifacts/task_artifact.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+from attrs import define, field
+from typing import TYPE_CHECKING
+from griptape.artifacts import ControlFlowArtifact
+
+if TYPE_CHECKING:
+    from griptape.tasks import BaseTask
+    from griptape.artifacts import BaseArtifact
+
+
+@define
+class TaskArtifact(ControlFlowArtifact):
```

Comment:
Not sure how I feel about this artifact. And in a fully declarative Workflow it may be impossible to return a Task object.

Can we just operate with Task ids in a TextArtifact?

Reply:
this is just used to return the `input` of the task. and it will always return the tasks because it pulls directly from the structure